src/PageContent/Calendar/DelivererGroupContent.js

```diff
@@ -0,0 +1,138 @@
+                    {this.props.accountType === AccountType.DELIVERER_GROUP ? (
+                        <h1 className="section-header">Student Deliverers</h1>
+                    ) : (
+                        <h1 className="section-header">Picking Up Donation</h1>
+                    )}
+                    <h2 className="organization">
+                        {this.props.delivererGroup}
+                    </h2>
+                    {!this.state.edit ? (
+                        <div className="content-details-wrapper">
+                            <p className="content-details">
+                                {this.state.deliverer1} {this.state.phone1}
+                            </p>
+                            <p className="content-details">
+                                {this.state.deliverer2} {this.state.phone2}
+                            </p>
+                        </div>
+                    ) : (
+                        <div className="content-details-wrapper">
+                            <form
+                                className="edit-dg"
+                                onSubmit={this.handleChange}
+                            >
+                                <div className="input-wrapper">
+                                    <input
+                                        className="content-details inline-details"
+                                        defaultValue={this.state.deliverer1}
+                                        name="deliverer1"
+                                        type="textyeah "
+                                    />
+                                    <input
+                                        className="content-details inline-details"
+                                        defaultValue={this.state.phone1}
+                                        name="phone1"
+                                        type="tel"
+                                    />
+                                    <input
+                                        className="content-details inline-details"
+                                        defaultValue={this.state.deliverer2}
+                                        type="text"
+                                        name="deliverer2"
+                                    />
+                                    <input
+                                        className="content-details inline-details"
+                                        defaultValue={this.state.phone2}
+                                        type="tel"
+                                        name="phone2"
+                                    />
+                                </div>
+
+                                <input
+                                    type="submit"
+                                    className="edit-button"
+                                    value="save"
+                                />
+                            </form>
```

Comment:
we'll actually also need to add the `email` as one of the inputs that they need to change (otherwise they'll only change the name and phone, but we'll keep the same old email).

Reply:
Only as a change input or as a displayed input as well?